tests/runner.py benchmark fails on incoming branch

I updated emscripten with `emsdk update`, then installed and activated `sdk-incoming-64bit` on windows. No errors while installing/activating. When i run `python tests\runner.py benchmark`, it fails with errors: (I've turned on EMCC_DEBUG)
```
C:\Program Files\Emscripten\emscripten\incoming>python tests\runner.py benchmark
DEBUG:root:Cache: acquiring multiprocess file lock to Emscripten cache
DEBUG:root:Cache: done
INFO:root:generating system asset: is_vanilla.txt... (this will be cached in "C:\Users\rschapha\.emscripten_cache\is_vanilla.txt" for subsequent builds)
DEBUG:root:Cache: acquiring multiprocess file lock to Emscripten cache
DEBUG:root:Cache: done
DEBUG:root:Cache: released multiprocess file lock to Emscripten cache
DEBUG:root:testing for asm.js target, because if not present (i.e. this is plain vanilla llvm, not emscripten fastcomp), we will use the wasm target instead (set EMCC_WASM_BACKEND to skip this check)
INFO:root: - ok
DEBUG:root:Cache: released multiprocess file lock to Emscripten cache
DEBUG:root:check tells us to use asm.js backend
WARNING:root:use EM_ALL_ENGINES=1 in the env to run against all JS engines, which is slower but provides more coverage
DEBUG:root:Checking JS engine ['C:/Program Files/Emscripten/node/4.1.1_64bit/bin/node']
Test suites:
['test_benchmark']
Running test_benchmark: (20 tests)
Running Emscripten benchmarks... [ including compilation | Wed Dec 07 12:06:44 2016 | em: commit 0efac40f87a8028dda26e467e2f19ed12822c84f | llvm: C:/Program Files/Emscripten/clang/fastcomp/build_incoming_vs2015_64/RelWithDebInfo/bin ]
test_conditionals (test_benchmark.benchmark) ...
Building native executable with command failed C:/Program Files/Emscripten/clang/fastcomp/build_incoming_vs2015_64/RelWithDebInfo/bin\clang++ -fno-math-errno c:/users/rschapha/appdata/local/temp\emscripten_temp\conditionals.cpp -o c:/users/rschapha/appdata/local/temp\emscripten_temp\conditionals.cpp.native -O2
Output: c:/users/rschapha/appdata/local/temp\emscripten_temp\conditionals.cpp:2:16: fatal error: 'stdio.h' file not found
      #include <stdio.h>
               ^
1 error generated.

None
ERROR
test_copy (test_benchmark.benchmark) ...
Building native executable with command failed C:/Program Files/Emscripten/clang/fastcomp/build_incoming_vs2015_64/RelWithDebInfo/bin\clang++ -fno-math-errno c:/users/rschapha/appdata/local/temp\emscripten_temp\copy.cpp -o c:/users/rschapha/appdata/local/temp\emscripten_temp\copy.cpp.native -O2
Output: c:/users/rschapha/appdata/local/temp\emscripten_temp\copy.cpp:2:16: fatal error: 'stdio.h' file not found
      #include <stdio.h>
               ^
1 error generated.

None
ERROR
test_corrections (test_benchmark.benchmark) ...
Building native executable with command failed C:/Program Files/Emscripten/clang/fastcomp/build_incoming_vs2015_64/RelWithDebInfo/bin\clang++ -fno-math-errno c:/users/rschapha/appdata/local/temp\emscripten_temp\corrections.cpp -o c:/users/rschapha/appdata/local/temp\emscripten_temp\corrections.cpp.native -O2
Output: c:/users/rschapha/appdata/local/temp\emscripten_temp\corrections.cpp:2:16: fatal error: 'stdio.h' file not found
      #include <stdio.h>
               ^
1 error generated.
.
.
.
```
similarly for other benchmarks. In the end it displays:
```
======================================================================
ERROR: test_conditionals (test_benchmark.benchmark)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Program Files\Emscripten\emscripten\incoming\tests\test_benchmark.py", line 443, in test_conditionals
    self.do_benchmark('conditionals', src, 'ok', reps=TEST_REPS*5)
  File "C:\Program Files\Emscripten\emscripten\incoming\tests\test_benchmark.py", line 194, in do_benchmark
    b.build(self, filename, args, shared_args, emcc_args, native_args, native_exec, lib_builder, has_output_parser=output_parser is not None)
  File "C:\Program Files\Emscripten\emscripten\incoming\tests\test_benchmark.py", line 82, in build
    shutil.move(filename + '.native', final)
  File "C:\Program Files\Emscripten\python\2.7.5.3_64bit\lib\shutil.py", line 301, in move
    copy2(src, real_dst)
  File "C:\Program Files\Emscripten\python\2.7.5.3_64bit\lib\shutil.py", line 130, in copy2
    copyfile(src, dst)
  File "C:\Program Files\Emscripten\python\2.7.5.3_64bit\lib\shutil.py", line 82, in copyfile
    with open(src, 'rb') as fsrc:
IOError: [Errno 2] No such file or directory: 'c:/users/rschapha/appdata/local/temp\\emscripten_temp\\conditionals.cpp.native'

======================================================================
ERROR: test_copy (test_benchmark.benchmark)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Program Files\Emscripten\emscripten\incoming\tests\test_benchmark.py", line 362, in test_copy
    self.do_benchmark('copy', src, 'sum:')
  File "C:\Program Files\Emscripten\emscripten\incoming\tests\test_benchmark.py", line 194, in do_benchmark
    b.build(self, filename, args, shared_args, emcc_args, native_args, native_exec, lib_builder, has_output_parser=output_parser is not None)
  File "C:\Program Files\Emscripten\emscripten\incoming\tests\test_benchmark.py", line 82, in build
    shutil.move(filename + '.native', final)
  File "C:\Program Files\Emscripten\python\2.7.5.3_64bit\lib\shutil.py", line 301, in move
    copy2(src, real_dst)
  File "C:\Program Files\Emscripten\python\2.7.5.3_64bit\lib\shutil.py", line 130, in copy2
    copyfile(src, dst)
  File "C:\Program Files\Emscripten\python\2.7.5.3_64bit\lib\shutil.py", line 82, in copyfile
    with open(src, 'rb') as fsrc:
IOError: [Errno 2] No such file or directory: 'c:/users/rschapha/appdata/local/temp\\emscripten_temp\\copy.cpp.native'
.
.
.
```
etc for others too, with
```
Ran 20 tests in 4.912s

FAILED (errors=15)
```
Any insights?

--- a/tests/test_benchmark.py
+++ b/tests/test_benchmark.py
@@ -68,8 +68,8 @@ class NativeBenchmarker(Benchmarker):
     if lib_builder: native_args = native_args + lib_builder(self.name, native=True, env_init={ 'CC': self.cc, 'CXX': self.cxx })
     if not native_exec:
       compiler = self.cxx if filename.endswith('cpp') else self.cc
-      cmd = [compiler, '-fno-math-errno', filename, '-o', filename+'.native'] + self.args + shared_args + native_args
-      process = Popen(cmd, stdout=PIPE, stderr=parent.stderr_redirect)
+      cmd = [compiler, '-fno-math-errno', filename, '-o', filename+'.native'] + self.args + shared_args + native_args + get_clang_native_args()
+      process = Popen(cmd, stdout=PIPE, stderr=parent.stderr_redirect, env=get_clang_native_env())
       output = process.communicate()
       if process.returncode is not 0:
         print >> sys.stderr, "Building native executable with command failed", ' '.join(cmd)
@@ -132,12 +132,16 @@ process(sys.argv[1])
 # Benchmarkers
 try:
   benchmarkers_error = ''
-  benchmarkers = [
-    NativeBenchmarker('clang', CLANG_CC, CLANG),
-    JSBenchmarker('sm-asmjs', SPIDERMONKEY_ENGINE, ['-s', 'PRECISE_F32=2']),
-    JSBenchmarker('sm-wasm',  SPIDERMONKEY_ENGINE, ['-s', 'BINARYEN=1', '-s', 'BINARYEN_METHOD="native-wasm"', '-s', 'BINARYEN_IMPRECISE=1']),
-    JSBenchmarker('v8-wasm',  V8_ENGINE,           ['-s', 'BINARYEN=1', '-s', 'BINARYEN_METHOD="native-wasm"', '-s', 'BINARYEN_IMPRECISE=1']),
-  ]
+  benchmarkers = [NativeBenchmarker('clang', CLANG_CC, CLANG)]
+  if SPIDERMONKEY_ENGINE and SPIDERMONKEY_ENGINE[0]:
+    benchmarkers += [
+      JSBenchmarker('sm-asmjs', SPIDERMONKEY_ENGINE, ['-s', 'PRECISE_F32=2']),
+      JSBenchmarker('sm-wasm',  SPIDERMONKEY_ENGINE, ['-s', 'BINARYEN=1', '-s', 'BINARYEN_METHOD="native-wasm"', '-s', 'BINARYEN_IMPRECISE=1'])
+    ]
+  if V8_ENGINE and V8_ENGINE[0]:
+    benchmarkers += [
+      JSBenchmarker('v8-wasm',  V8_ENGINE,           ['-s', 'BINARYEN=1', '-s', 'BINARYEN_METHOD="native-wasm"', '-s', 'BINARYEN_IMPRECISE=1'])
+    ]
 except Exception, e:
   benchmarkers_error = str(e)
   benchmarkers = []

